frontend/src/components/app/ErrorScreen.js

```diff
@@ -3,10 +3,15 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { checkLoginRequest } from '../../api/login';
 import { CONNECTION_ERROR_RETRY_INTERVAL_MILLIS } from '../../constants/Constants';
+import {
+  getCurrentActiveLocale,
+  extractLanguageFromLocale,
+} from '../../utils/locale';
 class ErrorScreen extends Component {
   constructor(props) {
     super(props);
     this.intervalId = null;
+    this.activeLang = extractLanguageFromLocale(getCurrentActiveLocale());
```

Comment:
instead of `extractLanguageFromLocale(getCurrentActiveLocale())` 
i would suggest to introduce getCurrentActiveLanguage() function.